From 6f467772331be1aa59a0d487c9ed0b944c0b5fa2 Mon Sep 17 00:00:00 2001
From: Gao Pan <pandy.gao@nxp.com>
Date: Mon, 24 Jul 2017 10:01:01 +0800
Subject: [PATCH 2186/5242] MLK-16063-1 arm64: dts: disable mlb in default dts

commit  a5f2b3e161dbd030fd63329b903c7d36b2dc105b from
https://source.codeaurora.org/external/imx/linux-imx.git

mlb has pin confict with ESAI. So this patch disable mlb in dts.

Signed-off-by: Gao Pan <pandy.gao@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
index 6dc7235..33a3ce9 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
@@ -479,7 +479,7 @@
 &mlb {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_mlb>;
-	status = "okay";
+	status = "disabled";
 };
 
 &usdhc1 {
-- 
1.7.9.5

